vignettes/r-markdown.Rmd

```diff
@@ -29,12 +29,12 @@ Behind the scenes, the engine relies on RStan to compile the model code into an
 in-memory `stanmodel`, which is assigned to a variable with the name given by
 the `output.var` chunk option. For example:
 
-````{verbatim}
-```{stan, output.var="model"}
+````markdown
+`r ''````{stan, output.var="model"}
```

Comment:
The `{verbatim}` chunks stopped working (it was attempting to evaluate the chunks) so I had to replace them with this other strategy for verbatim chunks that I found here: 

https://support.rstudio.com/hc/en-us/articles/360018181633-Including-verbatim-R-code-chunks-inside-R-Markdown